src/Form/Form.php

```diff
@@ -101,12 +101,23 @@ public function showForm($id, array $options = [])
         // We will be editing forms from this page
         echo Html::script("js/form_editor_controller.js");
 
+        $types_manager = QuestionTypesManager::getInstance();
+        $js_files = [];
+        foreach ($types_manager->getQuestionTypes() as $type) {
+            foreach ($type::loadJavascriptFiles() as $file) {
+                if (!in_array($file, $js_files)) {
+                    $js_files[] = $file;
+                    echo Html::script($file);
```

Comment:
`$js_files` should be passed to the template and rendered using a code like this one:
```
   {% for js_file in js_files %}
      <script src="{{ js_path(js_file) }}"></script>
   {% endfor %}
```